update-readmes.sh

```diff
@@ -8,6 +8,7 @@
 #   from the given github repo. If that branch is NOT master, then the
 #   branch name will be appended to the local readme file name.
 (cat <<LIST_END
+strongloop loopback-connector-mysql master
```

Comment:
nitpick: `strongloop loopback-connector-couchdb2 master` is also supported

Reply:
Let's add it after this one lands.